### ISSUE TITLE
[Code health] Rename `ground` folder back to `app`

### ISSUE THREAD

**Issue by gino-m** [google/ground-android#1389]
The path `ground-android/ground` is slightly confusing, and seems to imply the level about it is not actually part of `ground`.

Perhaps the default `app` would be more appropriate here?

@scolsen @JSunde @shobhitagarwal1612  for thoughts.

**Comment by scolsen** [comment]
app sgtm

**Comment by gino-m** [comment]
@JSunde @shobhitagarwal1612 Are you ok with this change as well?

**Comment by shobhitagarwal1612** [comment]
Yes. This seems ok to me as well

**Comment by Sachin-2206** [comment]
SIr may I help you in this ?

**Comment by gino-m** [comment]
Hi @Sachin-2206, thanks for offering to help!

Can you kindly sign https://cla.developers.google.com/clas before proceeding?

**Comment by EmmanuelDav** [comment]
@gino-m I would love to handle this issue if it's free

**Comment by Sachin-2206** [comment]
Done.

**Comment by Sachin-2206** [comment]
Yes it's free

**Comment by Sachin-2206** [comment]
@gino-m Sir I want to know that I just rename the 'ground' folder name to 'app' ?

**Comment by Sachin-2206** [comment]
@gino-m I just need a little help to know that what I exactly have to do in this issue. May you help me please ?

**Comment by EmmanuelDav** [comment]
Done

**Comment by gino-m** [comment]
> @gino-m Sir I want to know that I just rename the 'ground' folder name to 'app' ?

@Sachin-2206, yes, that's correct. The path to the module will also need to be updated in relevant config files (*.gradle, *.yaml).

@EmmanuelDav This issue was already assigned. Please wait for issues to be assigned to you before proceeding. Thanks!

**Comment by EmmanuelDav** [comment]
Ok. My apologies

**Comment by Sachin-2206** [comment]
@gino-m Sir I rename ground folder name to  app folder name. and I also rename the package name of all files of ground folder to app. Is it correct or should I change something else. I will send the sample or screenshot , check it and tell me as soon as possible.
![Screenshot (1)](https://user-images.githubusercontent.com/121143127/211539892-ea4fbef4-bc37-4839-b8fd-f51d8cccc7ca.png)

**Comment by gino-m** [comment]
> @gino-m Sir I rename ground folder name to  app folder name. and I also rename the package name of all files of ground folder to app. Is it correct or should I change something else. I will send the sample or screenshot , check it and tell me as soon as possible.
> ![Screenshot (1)](https://user-images.githubusercontent.com/121143127/211539892-ea4fbef4-bc37-4839-b8fd-f51d8cccc7ca.png)
> 

The package name does not need to change, only the module folder name at project root level. Thanks!

**Comment by devedroy** [comment]
Hello @Sachin-2206 ,
Are you still working on this issue?
If not, I would love to take this up.
Thankyou.

**Comment by gino-m** [comment]
@devedroy Sorry for late reply, still interested in taking this one?

**Comment by devedroy** [comment]
@gino-m Thanks for the reply.
Yes, I would love to take up this issue.

**Comment by gino-m** [comment]
> @gino-m Thanks for the reply. Yes, I would love to take up this issue.

Please go ahead!

**Comment by gino-m** [comment]
@devedroy Let me know if you're still interested and we can reassign.
